.github/workflows/push-tag-create-release.yml

```diff
@@ -102,7 +102,7 @@ jobs:
 
       # Only creates GH releases for client and build-tools releases.
       - name: Create GH release
-        if: fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'client' || fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'build-tools'
+        if: fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'client' || fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'build-tools' || fromJson(env.RELEASE_JSON).packageOrReleaseGroup == 'server'
```

Comment:
Thoughts on including this change @tylerbutler?

Reply:
Probably best to do that separately after we've validated the other changes.